config.default.yaml

```diff
@@ -629,15 +629,7 @@ solving:
     track_iterations: false
     min_iterations: 4
     max_iterations: 6
-    keep_shadowprices:
-    - Bus
-    - Line
-    - Link
-    - Transformer
-    - GlobalConstraint
-    - Generator
-    - Store
-    - StorageUnit
```

Comment:
Is this kept by default now? Or what was the reason to remove?

Reply:
Yes, kept by default :)